frontend/services.yml

```diff
@@ -19,6 +19,8 @@ services:
     command: sh -c "npm i && npm run dev"
     environment:
       - NEXT_PUBLIC_SUBLINKS_API_BASE_URL=sublinks:8080
+      - NEXT_PUBLIC_SUBLINKS_API_BASE_PUBLIC_URL=localhost
```

Comment:
@kgilles ideally this shouldn't be needed but rather default to "the same hostname that the browser URL has". Probably still good to allow it to be overrided though in case someone splits the API out to run on a subdomain